lab-David/test/read-file.test.js

```diff
@@ -0,0 +1,49 @@
+//4f6e65204f6e6520
+//54776f2054776f20
+//5468726565205468
+'use strict';
+const assert = require('assert');
+const read = require('../lib/read-file');
+
+describe('read-file.js', function(){
+  describe('#readData()', function(){
+    it('should return the first 8 bytes of one.txt in hex format', function(done){
+      read.readData('one.txt', function(err, parsedData){
+        assert.ifError(err);
+        assert.strictEqual(parsedData, '4f6e65204f6e6520');
+
+        done();
+      });
+    });
+    /* it('should return an error if given a bad file', function (done) {
+      read.readData('one.txt', function(err, parsedData){
+        assert.notStrictEqual(err, null);
+        assert.strictEqual(parsedData, undefined);
+
+        done();
+      });
+    }); */
```

Comment:
We like to delete commented code once we're sure we don't need it. Or, if you want to keep the test but don't know how to make it pass yet, you can [`skip()` it](https://mochajs.org/#inclusive-tests).